client/app/components/BigMessage.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 
 function BigMessage({ message, icon, children, className }) {
   return (
-    <div className={"big-message p-15 text-center " + className}>
+    <div className={"big-message p-15 text-center " + className} aria-live="assertive">
```

Comment:
Oriented by [this](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/hiding-and-updating-content).